tool/live-monitor.js

```diff
@@ -0,0 +1,166 @@
+const Web3 = require('web3');
+const { getBridgeTransactionByTxHash } = require('../index');
+const networkParser = require('./network-parser');
+const Bridge = require('@rsksmart/rsk-precompiled-abis').bridge;
+const util = require('util');
+
+const DEFAULT_CHECK_EVERY_MILLIS = 1_000;
+
+const defaultCommandsValues = {
+    fromBlock: 'latest',
+    methods: [],
+    events: [],
+    pegout: false,
+    pegin: false,
+    network: 'testnet',
+    checkEveryMillis: DEFAULT_CHECK_EVERY_MILLIS,
+};
+
+const getParsedCommands = () => {
+    const commands = process.argv.filter(command => command.startsWith('--'))
+    .reduce((commands, command) => {
+        if(command.startsWith('--fromblock')) {
+            commands.fromBlock = command.slice(command.indexOf('=') + 1);
+        } else if(command.startsWith('--methods') || command.startsWith('--events')) { // Parsing commands that include an array
+            commands['methods'] = JSON.parse(command.slice(command.indexOf('=') + 1).replaceAll("'", '"'));
+        } else if(command.startsWith('--network')) {
+            commands.network = command.slice(command.indexOf('=') + 1);
+        } else if(command.startsWith('--pegout') || command.startsWith('--pegin')) {
+            const parsedCommand = command.replace('--', '');
+            commands[parsedCommand] = true;
+        } else if(command.startsWith('--checkeverymillis')) {
+            commands.checkEveryMillis = Number(command.slice(command.indexOf('=') + 1));
+        } else {
+            throw new Error(`Command '${command}' not recognized`);
+        }
+        return commands;
+    }, defaultCommandsValues);
+    return commands;
+};
+
+const PEGOUT_METHOD_SIGNATURES = {
+    releaseBtc: '0x',
+    updateCollections: '0x0c5a9990',
+    addSignature: '0xf10b9c59'
+};
+
+const PEGIN_METHOD_SIGNATURES = {
+    registerBtcCoinbaseTransaction: '0xccf417ae',
+    registerBtcTransaction: '0x43dc0656',
+};
+
+function isAPegoutRelatedTransactionData(data) {
+    const methodSignature = data.slice(0, 10);
+    switch(methodSignature) {
```

Comment:
Maybe this can be replaced by a lambda?

Something like PEGOUT_METHOD_SIGNATURES.stream().anyMatch(methodSignature)

Reply:
I'm not sure about that. The switch case is really efficient. We can consider it to be constant time. But a lambda or something like that would be linear time. No big deal in this case, I know, but I think the switch case makes more sense, since we can see at a glance all the values that it is checking.

Reply:
The advantage of using a lambda in this case would be that if a new method is added to PEGOUT_METHOD_SIGNATURES there's no need to modify this switch statement and we don't risk forgetting about adding it here. What do you think?